api/src/models.d.ts

```diff
@@ -72,3 +72,158 @@ export interface ParticipantActivityForProgram {
   program_id: number;
   participant_activities: ParticipantActivityCompletionStatus[];
 }
+
+export interface Question {
+  assessment_question_id: number;
```

Comment:
This field does not appear in the data types Google Doc I'm referencing. What is this field, and what information does it reference?